requirements.txt

```diff
@@ -14,8 +14,8 @@ git+https://github.com/alphagov/digitalmarketplace-apiclient.git@19.15.1#egg=dig
 ## The following requirements were added by pip freeze:
 asn1crypto==0.24.0
 blinker==1.4
-boto3==1.9.134
-botocore==1.12.134
+boto3==1.9.139
+botocore==1.12.139
```

Comment:
It's very minor, but can we remove this bump from the commit? Just to keep things cleaner.